dap-launch.el

```diff
@@ -40,12 +40,13 @@ supported."
            (or (group
                 (or (: "//" (* nonl) eol)
                     (: "/*" (* (or (not (any ?*))
-                                   (: (+ ?*) (not (any ?/))))) (+ ?*) ?/)))
+                                   (: (+ ?*) (not (any ?/))))) (+ ?*) ?/)
+                    (: "," (group (* (any blank ?\C-j)) (any ?\} ?\])))))
```

Comment:
why not just use `space` instead of `blank and ?\n` only? In Windows, the newline characters can also include `\r`, sometimes even vertical space `\v` existed too. Wouldn't it safer to just check for all `space` group instead?

Reply:
Actually, this might be an issue, given that this isn't just JSON that we are parsing with its limited subset of whitespace, but JSON5, which supports more.

cc @jrwats

Reply:
Agreed. That `:blank:` misses other characters was an oversight on my part.  `:space:` won't match newline, however, that will still need `?\n` alongside it.

Reply:
https://github.com/emacs-lsp/dap-mode/pull/383

I'm really scratching my head about the `[[:space:]]*` vs `[[:space:]\n]*` repro...